chef/cookbooks/glance/templates/default/glance-cache.conf.erb

```diff
@@ -31,6 +31,9 @@ registry_host = <%= @registry_bind_host %>
 # Port the registry server is listening on
 registry_port = <%= @registry_bind_port %>
 
+<% # Missing in upstream template -%>
+registry_client_insecure = <%= @keystone_settings["insecure"] %>
```

Comment:
In https://github.com/crowbar/crowbar-openstack/pull/306, we had to change the recipe to define this variable